model/user/db.go

```diff
@@ -414,6 +414,25 @@ func ClearLoginCache(user gimlet.User) error {
 	return nil
 }
 
+// ClearUserSettings clears one user's settings
+func ClearUserSettings(userId string) error {
+	update := bson.M{"$unset": bson.M{SettingsKey: 1}}
+
+	u, err := FindOneById(userId)
```

Comment:
I'd rather not run this query since we just use userId for the update anyway.

Reply:
done